### DISCUSSION TITLE
Issue when following "installing and configuring VSCODE"

### DISCUSSION THREAD

**Discussion by orbitcoms** [Mair/esp32-course#71] [Course Qeustions] [answered]
Hi,

I am following along exactly with the steps in the "Environment Setup" -> "Installing and configuraging VsCode".

Everything worked the same as shown in the video until setting up the Espressif extension. (esp-idf was installed and trested ok).

When I open "ESP-IDF:Configure ESP_IF Extension", I do NOT get the "Use Existing Setup" option as shown in the video. I only see "Express" and "Advanced" options.

What causes this and how can I get VSCODE to display this option so I can use the ESP-IF aleady installed?

**Comment by Mair** [comment]
That's correct. I mention that you may not see it because you have not previously set it up.
see https://learnesp32.com/videos/1/3_installing%20and%20configuring%20vs%20code at 4:00

**Comment by orbitcoms** [comment, marked as answer]
Perfect, thanks Mair.
